src/core/server/index.ts

```diff
@@ -307,6 +310,8 @@ class Server {
     );
 
     logger.info({ port }, "now listening");
+
+    this.scheduledTasks = startCronJobs(this.mongo, this.tasks.mailer);
```

Comment:
Saving variables in `scheduledTasks` yet function is called `startConJobs`?

Reply:
Also, prefer object for variables rather than positional ones (requirements likely to expand).